src/components/Days-Catches/DisplayCatches.js

```diff
@@ -1,28 +1,74 @@
 import styled from 'styled-components';
+import DeleteModal from '../modal/DeleteModal';
 import CatchCard from './CatchCard';
 
-export default function DisplayCatches({ catches, onDelete }) {
+export default function DisplayCatches({
+  catches,
+  onDeleteCatch,
+  confirmDeleteCatch,
+  cancelDeleteCatch,
+  showCatchModal,
+}) {
   return (
-    <Catches>
-      {catches ? (
-        catches.map((data, tempId, _id) => (
-          <li key={tempId}>
-            <CatchCard data={data} onDelete={onDelete} />
-          </li>
-        ))
-      ) : (
-        <p>no catches yet</p>
+    <>
+      <Catches>
+        {catches ? (
+          catches.map((data, tempId, _id) => (
+            <li key={tempId}>
+              <CatchCard data={data} onDelete={onDeleteCatch} />
+            </li>
+          ))
+        ) : (
+          <p>no catches yet</p>
+        )}
+      </Catches>
+      {showCatchModal && (
+        <Container>
+          <Modal>
+            <Title>Are you sure?</Title>
+            <DeleteModal
+              text="catch"
+              confirmDelete={confirmDeleteCatch}
+              cancelDelete={cancelDeleteCatch}
+            />
+          </Modal>
+        </Container>
       )}
-    </Catches>
+    </>
   );
 }
 
 const Catches = styled.ul`
   display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
-  justify-content: center;
+  grid-template-columns: repeat(3, 1fr);
   gap: 10px;
   list-style: none;
   padding: 0;
   width: 100%;
 `;
+
+const Container = styled.div`
```

Comment:
Do you really need `Container` and `Modal`?  You could put this in one `div` element and check which stylings you don't need. 

And same here for the naming of `Modal` as mentioned above.